mantidproject/mantid
# Update Release Notes v4.2 Step2

**Reformat and add Images to Muon, Reflectometry and Sans Pages in the release notes for v4.2**



**To test:**

Build docs.html and check that links work, images show up correctly and all bullet points are neatly arranged!

refs #27100 


*This does not require release notes* because **fill in an explanation of why**
It is release notes!!!

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
